docs/source/python/ipc.rst

```diff
@@ -154,6 +154,73 @@ DataFrame output:
    df = pa.ipc.open_file(buf).read_pandas()
    df[:5]
 
+Efficiently Writing and Reading Arrow Arrays
```

Comment:
Say "record batches" instead of "arrays"?

Reply:
Maybe `Efficiently Writing and Reading Data` to keep it more general?
I mean, when reading in those examples we really don't use record batches and when writing it seems the record batches are the way through which you achieve the efficiency (chunking) more than the _goal_ itself.

Reply:
This is true, but the text above always talks about record batches specifically. That said, "Arrow data" would be fine to me (bare "data" is too general IMHO).

Reply:
:+1: